core/src/main/java/org/frankframework/util/flow/MermaidFlowGenerator.java

```diff
@@ -206,10 +208,11 @@ private boolean matchesTestClassPath(MetadataReader reader, MetadataReaderFactor
 	}
 
 	@Override
-	public void generateFlow(String xml, OutputStream outputStream) throws FlowGenerationException {
+	public void generateFlow(String xml, OutputStream outputStream, String name) throws FlowGenerationException {
 		try {
+			log.debug("Generating flow for the following XML: [{}]", xml);
 			String flow = generateMermaid(xml);
-
+			log.debug("Have the following Mermaid code to draw Configuration or Adapter [{}]: {}", name, flow);
```

Comment:
Dit is denk ik niet zo handig. Zowel de gehele adapter/configuratie xml loggen, en het meegeven van de naam zou niet moeten hoeven.

Reply:
Deze code had ik nodig om de unit tests te kunnen maken. Ik paste telkens mijn voorbeeld config aan. Daarna keek ik of het plaatje lukte. Vervolgens las ik de syntax 1 (oude syntax, input van de test) en de gegenereerder Mermaid (output) uit de log. Ik had toen ik aan het ontwikkelen was log level info omdat ik niet wou uitzoeken hoe ik debug logs kon zien. Toen de tests klaar waren heb ik de logs teruggezet naar level "debug". Normaal gesproken zou je ze dus niet moeten zien zodat de log compact blijft.

Reply:
Het was ook erg handig om de naam te hebben. Zo kan je makkelijk zoeken van welke config of adapter je de Mermaid ziet.

Reply:
Ik snap dat het voor het testen handig is, dan kan het bij de test er bijgezet worden. Dit moet niet in de productie code komen.

Reply:
Ik heb bijna alle log statements waar we het over hadden weggehaald, maar ik heb er één laten staan. Dat is bij het initialiseren van de MermaidFlowGenerator.java. Kan deze wel?

Reply:
Ik heb nog even gekeken naar de configuratie xml, naast dat deze in de console beschikbaar is, wordt deze ook gelogd in de configuration-log.

Reply:
Bedankt.